x-pack/plugins/apm/public/components/shared/license_prompt/index.tsx

```diff
@@ -8,41 +8,29 @@
 import { EuiButton, EuiCard, EuiTextColor } from '@elastic/eui';
 import React from 'react';
 import { i18n } from '@kbn/i18n';
+import { useApmPluginContext } from '../../../context/apm_plugin/use_apm_plugin_context';
 import { useKibanaUrl } from '../../../hooks/use_kibana_url';
 
 export interface LicensePromptProps {
   text: string;
-  showBetaBadge?: boolean;
 }
 
-export function LicensePrompt({
-  text,
-  showBetaBadge = false,
-}: LicensePromptProps) {
+export function LicensePrompt({ text }: LicensePromptProps) {
+  const {
+    plugins: { licenseManagement },
+  } = useApmPluginContext();
   const licensePageUrl = useKibanaUrl(
     '/app/management/stack/license_management'
   );
-
+  const manageLicenseURL = licenseManagement?.locator
+    ? licenseManagement?.locator?.useUrl({
+        page: 'dashboard',
+      })
+    : licensePageUrl;
```

Comment:
hi @sqren,
We talk about what will happen if `licenseManagement.locator` is undefined, but I'm actually not sure if we want to fall back to the `licensePageUrl = '/app/management/stack/license_management'`
what else we can do in this case? any suggestions?

Reply:
In the case where license management plugin is disabled we don't want to link to the license page, because it'll display a 404. In that case I think we can just not show the link. I don't think this will ever happen. If the license management is not enabled, the user ought to have something like an enterprise license, with access to everything.